lib/engine/game/g_18_ga.rb

```diff
@@ -27,17 +29,17 @@ def setup
         setup_company_price_50_to_150_percent
 
         # Place neutral tokens in the off board cities
-        neutral = Corporation.new(
+        @neutral = Corporation.new(
```

Comment:
why make these instance variables? you never use them

Reply:
Removed